libs/blocks/faas/utils.js

```diff
@@ -248,7 +248,7 @@ const beforeSubmitCallback = () => {
       }),
     })
       .catch((error) => {
-        window.lana.log('AA Sandbox Error:', error);
+        window.lana.log(`AA Sandbox Error: ${error.reason || error.error || error.message || error}`, { tags: 'errorType=info,module=faas' });
```

Comment:
Why is the `error.reason || error.error || error.message || error` pattern needed here, but not in other places?